Comp2024/src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -147,16 +147,12 @@ public void periodic( )
     }
   }
 
-  // public PathPlannerPath driveToSpeaker(CommandSwerveDrivetrain drivetrain)
-  // {
-  //   in progress
-
-  //   Pose2d speakerEndGoal = (1.4,5.52,0)
-  //   PathConstraints speakerConstraints = 
-
-  //   PathPlannerPath driveToSpeaker = new PathPlannerPath (// odometry current pose, constraints, end goal);
-  //   return driveToSpeaker;
+  public Command driveWithLL(CommandSwerveDrivetrain drivetrain, Pose2d pose)
+  {
```

Comment:
As mentioned above, the constraints are really a "constant" value. And the parameters are possibly already defined for our robot (max velocity, angular rotation, etc.).

The method name could tell more about what it is doing (drivePathToPose) rather than how it is doing it (...withLL). This is a fundamental characteristic of code readability. (good naming is harder than it seems)